src/msg/async/Stack.cc

```diff
@@ -37,9 +37,7 @@
 std::function<void ()> NetworkStack::add_thread(Worker* w)
 {
   return [this, w]() {
-      char tp_name[16];
-      sprintf(tp_name, "msgr-worker-%u", w->id);
-      ceph_pthread_setname(pthread_self(), tp_name);
+      rename_thread(w->id);
```

Comment:
ideally, we should have two commits:

1. to do the refactory, which extract this part into a virtual method named `rename_thread()`
2. to fix the behavior of DPDK backend.